portal-web/test/functional/com/liferay/portalweb/macros/LiferayEvent.macro

```diff
@@ -0,0 +1,10 @@
+definition {
+
+	macro waitForPortletReady {
+		WaitForLiferayEvent(
```

Comment:
Not necessarily to do now, but it would be great if you could assess something in the event payload other than `eventName` to make sure you don't get 2 `portletReady` events and mistakenly pick the one that is not related to the thing you're waiting for.

Reply:
follow up ticket: https://issues.liferay.com/browse/LRQA-65793